app/indexer/detect_open.py

```diff
@@ -3,11 +3,16 @@
 
 
 def open_site(url):
-    '''Checking for wikipedia or SO page'''
-    for i in ["wikipedia.org", "stackoverflow.com"]:
-        if i in url:
-            return True
-    return False
+    with open("opensites.txt",'r') as file: #Using urls specified in opensites.txt
+        '''Checking for wikipedia or SO page'''
+        content = file.readlines()
+        urls = []
+        for i in content:
+            urls.append(i[0:int(len(i)-1)])
```

Comment:
Would one of the solutions in https://stackoverflow.com/questions/12330522/reading-file-without-newlines-in-python be cleaner?

Reply:
I agree with @asdofindia . I personally use file.read().splitlines(). But whichever you prefer...